var/spack/repos/builtin/packages/sprng/package.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Sprng(AutotoolsPackage):
+    """SPRNG: A Scalable Library For Pseudorandom Number Generation
+       Sprng is a distributed process-aware random number generator that
+       avoids correlations in random number sequences across processes.
+    """
+
+    maintainers = ['kayarre']
+
+    homepage = "http://www.sprng.org"
+    url      = "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
+
+    version('5.0', sha256='9172a495472cc24893e7489ce9b5654300dc60cba4430e436ce50d28eb749a66')
+
+    variant('mpi', default=True, description='Enable MPI support')
+    variant('fortran', default=False, description='Enable Fortran support')
+
+    depends_on('mpi', when='+mpi')
+
+    def url_for_version(self, version):
+        if version.version == (5, 0):
+            return "http://www.sprng.org/Version5.0/sprng5.tar.bz2"
```

Comment:
This won't work for other versions. What about:
```suggestion
        url = "http://www.sprng.org/Version{0}/sprng{1}.tar.bz2"
        return url.format(version, version.up_to(1))
```